client/my-sites/themes/controller.jsx

```diff
@@ -24,6 +30,7 @@ function makeElement( ThemesComponent, Head, store, props ) {
 			canonicalUrl={ props.canonicalUrl }
 			image={ props.image }
 			tier={ props.tier || 'all' }>
+			<DocumentHead title={ BASE_TITLE } />
```

Comment:
Oops, should've just used `props.title` here instead of the `const`. Oh well, didn't spot it when reviewing either. Will fix as part of #7224.